controllers/handler/nodenetworkconfigurationpolicy_controller.go

```diff
@@ -185,7 +184,7 @@ func (r *NodeNetworkConfigurationPolicyReconciler) Reconcile(ctx context.Context
 	if r.shouldIncrementUnavailableNodeCount(instance, previousConditions) {
 		err = r.incrementUnavailableNodeCount(instance)
 		if err != nil {
-			if apierrors.IsConflict(err) {
+			if apierrors.IsConflict(err) || errors.Is(err, node.MaxUnavailableLimitReachedError{}) {
```

Comment:
we don't need `IsConflict` anymore right ?

Reply:
In case we happen to get a conflict in all attempts, we can still get a conflict error, can't we?

Reply:
Right, the conflict error is returned as stated here
https://github.com/kubernetes/client-go/blob/8bffd27f0ac161b10909e7593264cb87a616c410/util/retry/util.go#L62-L64